docs/index.zh.md

```diff
@@ -0,0 +1,8 @@
+---
+hide:
+  - navigation
+---
+
+# Roll Off Club
```

Comment:
Note this file is named `docs/index.zh.md`, so this is where you put the chinese translation of the english version of the file at `docs/index.md`.